### PR TITLE
feat: change heartbeat and keepalive option connection

### PR DESCRIPTION
**Pull request by devmanu**
On change les deux paramètres de connexion:
 - keepalive => true
 - heartbeat => 1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea
```
Comment (stof) [inline .gitignore]: should not be there. It is related to your setup, and so belongs to your computer. See https://help.github.com/articles/ignoring-files/

Comment (devmanu) [reply .gitignore]: oups sorry bad place

**Comment by stof** [comment]
@devmanu why closing the PR ?

**Comment by devmanu** [comment]
because is for my fork. I made a mistake :)

**Comment by devmanu** [comment]
In fact we want change hearbeat and keepalive value and it is not possible now

**Pull request closed without merging**